### DISCUSSION TITLE
Support fixed interval feature

### DISCUSSION THREAD

**Discussion by tkboon82** [jobrunr/jobrunr#459] [Q&A]
Does Jobrunr support fixed interval feature ?

**Comment by rdehuyss** [comment]
Yes, see [the docs](https://www.jobrunr.io/en/documentation/background-methods/recurring-jobs/).

**Reply by tkboon82** [reply]
Sorry my bad, I mean does Jobrunr support fixed delay interval ?

**Reply by rdehuyss** [reply]
No.
